src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <the-nav />
+    <the-nav @toggle-nav="handleBody($event)" />
```

Comment:
This change is unintended, it should be removed from this PR (you can literally create a new commit where it's deleted.

Reply:
Same here.

Also deleted the relevant changes in the TheNav component.